cmd/operator-sdk/bundle/validate.go

```diff
@@ -66,19 +66,19 @@ just locally.
 and validate them, assuming a bundle for 'test-operator' version v0.1.0 exists at
 <project-root>/deploy/olm-catalog/test-operator/manifests:
 
-  # Generate manifests locally.
-  $ operator-sdk bundle create --generate-only
+  # Generate manifests and metadata locally.
+  $ operator-sdk generate bundle
 
   # Validate the directory containing manifests and metadata.
   $ operator-sdk bundle validate ./deploy/olm-catalog/test-operator
```

Comment:
Is this still the correct default path for the bundle?

Reply:
There is no default bundle path, but this should be updated. Ditto below.